pkgs/desktops/gnome-2/desktop/gtksourceview/default.nix

```diff
@@ -9,6 +12,21 @@ stdenv.mkDerivation rec {
     url = "mirror://gnome/sources/gtksourceview/2.10/${name}.tar.bz2";
     sha256 = "c585773743b1df8a04b1be7f7d90eecdf22681490d6810be54c81a7ae152191e";
   };
-  buildInputs = [pkgconfig atk cairo glib gtk pango libxml2Python perl intltool
-    gettext];
+
+  patches = optionals stdenv.isDarwin [
+    ./0001-Change-IgeMacIntegration-to-GtkOSXApplication.patch
```

Comment:
Thanks for the hint, was not yet aware of it. I'll try to use it for both patches.